samples/bluetooth/direct_test_mode/src/main.c

```diff
@@ -82,15 +81,14 @@ void main(void)
 		/* This is the second byte read; combine it with the first and
 		 * process command.
 		 */
-		if (current_time >
-			(msb_time + MAX_ITERATIONS_NEEDED_FOR_NEXT_BYTE)) {
```

Comment:
Please remove this define and its description if its no longer needed

Reply:
Yes right. Removed